cmake/catkin_package.cmake

```diff
@@ -374,6 +377,15 @@ function(_catkin_package)
       endif()
     endforeach()
 
+    set(PKG_COMPILE_OPTIONS ${${PROJECT_NAME}_COMPILE_OPTIONS})
+    separate_arguments(PKG_COMPILE_OPTIONS)
+    separate_arguments(PROJECT_DEFINES)
```

Comment:
In the context of the related discussion I should clarify that I assume the arguments are being passed like this (same as e.g. `DEPENDS`):

`catkin_package(DEFINED "something" "something-else")`